vendor/github.com/jesseduffield/gocui/tcell_driver.go

```diff
@@ -417,6 +420,11 @@ func (g *Gui) pollEvent() GocuiEvent {
 			Type:    eventFocus,
 			Focused: tev.Focused,
 		}
+	case *tcell.EventPaste:
+		return GocuiEvent{
+			Type:  eventPaste,
+			Start: tev.Start(),
```

Comment:
Is the idea that there's two paste events: one for starting the paste and one for finishing the paste, and if it's the first kind then Start will be true? If so it would be good to add a comment to that affect against the GocuiEvent struct

Reply:
Yes, that's true. I added a comment in 76924d8982fa, is that enough? Feel free to push fixups if you want to further improve it.